pysac/mhs_atmosphere/parameters/model_pars.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Dec 11 17:45:48 2014
+
+@author: sm1fg
+"""
+
+import numpy as np
+import sys
+
+def get_model(model, 
```

Comment:
I would separate this function into two parts.

I would rename this just `get_coords` which is model independent. I would then just put all the dictionaries of model parameters in this file (not in a function) and then in the top level script you could do

```python
from pysac.mhs_atmsophere.parameters.models import hmi_model as my_model
```
and then just use `my_model` as the dictionary.